### ISSUE TITLE
Hardware Acceleration Support for Ryzen Mobile CPUs?

### ISSUE THREAD

**Issue by DennisFury** [blakeblackshear/frigate-hass-addons#101]
I have the AMD NUC that has a Ryzen 7 4600U (https://www.amd.com/en/product/9096).

How do we get frigate HASS addon working w/ hardware acceleration for video decoding?  

Home Assistant 2023.2.5
Supervisor 2023.01.1
Operating System 9.5

I found this thread https://github.com/blakeblackshear/frigate/issues/311 , and the solution seems to be to install mesa-va-drivers and update an environment variable ; however HA OS doesn't seem to have the apt package manager to install mesa-va-drivers .  I also connected to the docker container that homeassistant runs in HA OS and no apt package manager either.

Anyone know how to get this setup?

**Comment by NickM-27** [comment]
This isn't supported for HA OS addon installs, there's no way to install the driver and the driver is required

**Comment by DennisFury** [comment]
Thanks Nick, would my path forward be to go to the HA OS https://github.com/home-assistant/operating-system and ask that they build in support for the driver?

**Comment by NickM-27** [comment]
Yes, you can request that it is included, I get the feeling they may push back though
